docs/release.md

```diff
@@ -0,0 +1,53 @@
+# opentelemetry-collector-releases Release Procedure
+
+This document describes the steps to follow when doing a new release of the
+opentelemetry-collector-releases repository. This release depends on the [opentelemetry-collector release][1] and the [opentelemetry-collector-contrib release][2] repositories.
+
+For general information about all Collector repositories release procedures, see the
+[opentelemetry-collector release process documentation][1].
+
+## Releasing opentelemetry-collector-releases
+
+0. Ensure that the [opentelemetry-collector][1] and [opentelemetry-collector-contrib][2] release procedures have been followed and a new
+   opentelemetry-collector and opentelemetry-collector-contrib version have been released. The opentelemetry-collector-releases release
+   should be done after both of these releases.
+1. Run the GitHub Action workflow "[Update Version in Distributions and Prepare PR](https://github.com/open-telemetry/opentelemetry-collector-releases/actions/workflows/update-version.yaml)" which will update the minor version automatically (e.g. v0.116.0 -> v0.117.0) or manually provide a new version if releasing a bugfix or skipping a version. Select "create pr" option.
+   -  🛑 **Do not move forward until this PR is merged.** 🛑
+2. Check out main and ensure it has the "Update version from ..." commit in your local
+   copy by pulling in the latest from
+   `open-telemetry/opentelemetry-collector-releases`. Assuming your upstream
+   remote is named `upstream`, you can try running:
+   - `git checkout main && git fetch upstream && git rebase upstream/main`
+3. Create a tag for the new release version by running:
+   
+   ⚠️ If you set your remote using `https` you need to include `REMOTE=https://github.com/open-telemetry/opentelemetry-collector-releases.git` in each command. ⚠️
+   
+   - `make push-tags TAG=v0.85.0`
+4. Wait for the new tag build to pass successfully.
+5. Ensure the "Release Core", "Release Contrib", "Release k8s", and "Builder - Release" actions pass, this will
```

Comment:
```suggestion
5. Ensure the "Release Core", "Release Contrib", "Release k8s", "Release OTLP", "Release Builder" and "Release OpAMP Suporvisor" actions pass, this will
```